hw/ip_templates/ac_range_check/rtl/ac_range_check.sv.tpl

```diff
@@ -334,41 +333,24 @@ module ${module_instance_name}
   // Interrupt Notification Logic
   //////////////////////////////////////////////////////////////////////////////
 
-  logic deny_cnt_threshold_reached_d, deny_cnt_threshold_reached_event;
-
-  // Create a threshold event when the deny counter reaches the configured threshold
-  assign deny_cnt_threshold_reached_d = deny_cnt > reg2hw.log_config.deny_cnt_threshold.q;
-  prim_edge_detector u_edge_threshold_reached (
-    .clk_i             ( clk_i                            ),
-    .rst_ni            ( rst_ni                           ),
-    .d_i               ( deny_cnt_threshold_reached_d     ),
-    .q_sync_o          (                                  ),
-    .q_posedge_pulse_o ( deny_cnt_threshold_reached_event ),
-    .q_negedge_pulse_o (                                  )
-  );
-
-  prim_edge_detector u_edge_intr_state (
-    .clk_i             ( clk_i               ),
-    .rst_ni            ( rst_ni              ),
-    .d_i               ( reg2hw.intr_state.q ),
-    .q_sync_o          (                     ),
-    .q_posedge_pulse_o (                     ),
-    .q_negedge_pulse_o ( intr_state_cleared  )
-  );
+  // Create the IRQ condition when the deny counter reaches the configured threshold
```

Comment:
Nit: I'd probably phrase this as "is above the configured threshold", to avoid it sounding like we're talking about the moment that we get there.